src/Generator/JsonGenerator.php

```diff
@@ -17,42 +18,55 @@ class JsonGenerator
 {
     use GeneratorTrait;
 
-    public function generateJson(
-        array $documents,
-        BuildContext $buildContext,
-        ProgressBar $progressBar
-    ) {
-        $environments = $this->extractEnvironments($documents);
+    public function generateJson(array $documents, ProgressBar $progressBar)
+    {
+        $this->extractEnvironmentsAndCachedMetas($documents);
+
+        if (!$this->buildContext->getDisableCache()) {
+            $cachedMetasLoader = new CachedMetasLoader();
+            $cachedMetasLoader->loadCachedMetaEntries($this->buildContext->getHtmlOutputDir(), $metas = new Metas());
+        }
```

Comment:
Is this needed now? Can't we use `$this->metas` after calling `extractEnvironmentsAndCachedMetas()`?

Reply:
yes that's a mistake, sorry

Reply:
fixed